src/newcharacter.cpp

```diff
@@ -784,6 +797,9 @@ Scenarios and professions affect skill point pool" ) ),
             retval = tab_direction::FORWARD;
         } else if( action == "QUIT" && query_yn( _("Return to main menu?") ) ) {
             retval = tab_direction::QUIT;
+        } else if (action == "HELP_KEYBINDINGS") {
+            // Need to redraw since the help window obscured everything.
+            draw_tabs(w, _("POINTS"));
```

Comment:
Check this line and all other lines in your PR: add a whitespace after ```(``` and before ```)```, remove space from ```if (``` to get ```if( ```. We have a special tool for that restyling (```astyle```) but it's okay to format small PRs by hands.